lazy_tensor_core/lazy_tensor_core/csrc/view_ops/opcode.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <torch/csrc/lazy/ts_backend/ts_node.h>
+
+namespace torch_lazy_tensors {
+namespace ir {
+namespace ops {
+
+// TODO: Allow other backend to define its own BaseNode, or
+// promote TsNode to Node.
+using BaseNode = torch::lazy::TsNode;
```

Comment:
ah.. here it is.  Maybe if you rename this file to ... ViewBase?  (and maybe even rename BaseNode to ViewBase?) it would become a little more discoverable without explanation